runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpClientFactory.java

```diff
@@ -1869,6 +1862,9 @@ private void cleanupEncodeSlotIfNecessary()
         private BudgetDebitor responseDeb;
         private long responseDebIndex = NO_DEBITOR_INDEX;
 
+        private long traceId;
+        private long authorization;
+
```

Comment:
Are these needed just to retain their values while queuing `beginEx` in slot, or do we need them for the lifetime of the exchange?

If only needed to survive queuing, then we should store them along with the `streamId` and `beginEx` in the slot entry instead.

Reply:
Yeah, good catch we only need them to survive queuing.